plugins/repository-azure/src/main/java/org/opensearch/repositories/azure/AzureStorageService.java

```diff
@@ -159,10 +162,27 @@ public Tuple<BlobServiceClient, Supplier<Context>> client(String clientName, BiC
         return new Tuple<>(state.getClient(), () -> buildOperationContext(azureStorageSettings));
     }
 
+    private StorageEndpoint getStorageBlobEndpoint(final AzureStorageSettings settings) {
+        try {
+            String endpointSuffix = settings.getEndpointSuffix();
+            if (!Strings.hasText(endpointSuffix)) {
+                endpointSuffix = "core.windows.net";
+            }
+            final URI primaryBlobEndpoint = new URI("https://" + settings.getAccount() + ".blob." + endpointSuffix);
```

Comment:
Please use `URI.create` - it will throw `IllegalArgumentException`, no need for try / catch block

```suggestion
            final URI primaryBlobEndpoint = URI.create("https://" + settings.getAccount() + ".blob." + endpointSuffix);
```

Reply:
fixed, thank you for the recommendation @reta.